src/components/inputs/TextField.js

```diff
@@ -361,6 +361,10 @@ export default class TextField extends BaseInput {
     return enableErrors && error;
   }
 
+  getErrorMessage() {
```

Comment:
Couldn't use the parent's original `getErrorMessage` since `generateStyles` getting called before the state created (in BaseComponent) and it produces error.